src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="MSBuild.Sdk.Extras/2.1.2">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0-ios;net6.0-android;net6.0-maccatalyst</TargetFrameworks>
+    <TargetFrameworks>net6.0-ios14.0;net6.0-android;net6.0-maccatalyst13.5</TargetFrameworks>
```

Comment:
No sure about these version numbers here?

Reply:
Do we need the specific iOS / macOS OS version numbers?

If not, let's leave them off for now since .NET MAUI only doesn't specify an OS:
```csharp
net6.0-ios;net6.0-maccatalyst;net6.0-android
```
https://github.com/dotnet/maui/blob/cadf5472f5d3d4a1201928b54c05a81a614fa3ba/Directory.Build.props#L8

Reply:
Yeah that's the funny thing. They don't specify it, but when I don't it starts complaining that I have to 🤷

Reply:
Ah - if we're getting build errors, then let's keep the OS numbers 💯 

According to the docs, .NET MAUI supports iOS 10+ and macOS 10+: https://docs.microsoft.com/dotnet/maui/supported-platforms

<img width="574" alt="image" src="https://user-images.githubusercontent.com/13558917/129396902-b31bb0c6-6117-4cf1-a192-72c2722ca3e2.png">

Supporting iOS 10+ is surprising to me. I didn't expect it to go that far back.

Reply:
I think we might need to dig into this a bit deeper though, because I think the things I set now are the _target_ versions, not the minimum versions.

Reply:
I [confirmed with the engineering team](https://teams.microsoft.com/l/message/19:989ffa44998147aca4ceaf7482967668@thread.skype/1628880426523?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=0056f60b-301f-43ac-bbcf-f356d3c42c92&parentMessageId=1628875764413&teamName=VS%20Client%20Experiences&channelName=MAUI%20%F0%9F%8C%BA&createdTime=1628880426523) that these are the current targets for Maui:
- iOS 10+
- macOS 10.13+

Reply:
> I think the things I set now are the target versions, not the minimum versions

These look to be the minimum versions. 

👇 Here's what I got when I ran `dotnet pack` to create a NuGet Package using `<TargetFrameworks>net6.0-ios14.0;net6.0-android;net6.0-maccatalyst13.5</TargetFrameworks>`:
<img width="717" alt="image" src="https://user-images.githubusercontent.com/13558917/129406758-1da44db7-bc5d-4fa4-b0ea-fc01ee94e3c5.png">

Reply:
That's the thing, I don't think that's the case. I think it's more like the Android way: there target version specifies which APIs you can use, but it's not the minimum version that you can run it on. Does that make sense?

Reply:
And here's the errors I got when I tried to add this NuGet Package to a new Maui App that was using lower OS versions:

### Maui Sample App Versions:
```xml
<TargetFrameworks>net6.0-ios13.0;net6.0-android29.0;net6.0-maccatalyst13.4</TargetFrameworks>
```

### NuGet Errors
<img width="1377" alt="image" src="https://user-images.githubusercontent.com/13558917/129407374-2f83bcdf-3e21-4652-9793-df23e3c84c90.png">

Reply:
## tl;dr

Here's what we need to set our `TargetFrameworks` to:

```xml
<TargetFrameworks>net6.0-ios10.0;net6.0-android21.0;net6.0-maccatalyst10.13</TargetFrameworks>
```

Reply:
> target version specifies which APIs you can use, but it's not the minimum version

It's both, right? Like targeting .NET Standard 2.0 means that I cannot use the `IAsyncEnumerable` because that is only available in .NET Standard 2.1. It also means that anybody targeting .NET Standard 2.0+ can use the library.

Reply:
What errors are we getting when we omit the iOS/Android/MacCatalyst version numbers?

After speaking with the engineering team, they recommend that we don't include version numbers.

Reply:
Got it! 

We need to remove both the OS version numbers AND `RuntimeIdentifier`, `InvariantGlobalization` and `UseInterpreter` from the csproj. I'll do that now 👍